test/core/keyTests.ts

```diff
@@ -0,0 +1,45 @@
+﻿///<reference path="../testReference.ts" />
+
+describe("DatasetKey", () => {
+  it("is passed to Dataset", () => {
+    let ds: Plottable.Dataset = new Plottable.Dataset();
```

Comment:
new to typescript - can you give some guidance on when it is appropriate or not appropriate to use explicit typing?